### PR TITLE
[CMake] Cleaning on the project and one scene

### PR DESCRIPTION
**Pull request by hugtalbot**
It would be nice to rename the GitHub repository as "Cosserat" (without "plugin.") .

What do you think about this PR @adagolodjo ?

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -113,21 +113,18 @@ if(SofaPython3Tools)
...
-    message("-- Python3 packages will be installed.")
-    set(SP3_PYTHON_PACKAGES_DIRECTORY "python3/site-packages")
-    # SP3_add_python_package(
-    #         SOURCE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/python3/cosserat
-    #         TARGET_DIRECTORY cosserat
-    # )
+    message("-- Python3 cosserat site-packages will be installed.")
+    SP3_add_python_package(
+        SOURCE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/python3/softrobots
+        TARGET_DIRECTORY softrobots
```
Comment (adagolodjo) [inline CMakeLists.txt:123]: `cosserat` instead of `softrobots`, right ?

**Comment by adagolodjo** [comment]
Hey @alxbilger, any suggestions on this? 
Tkx @hugtalbot

**Review by alxbilger — CHANGES_REQUESTED** [review]
you should also have a look in `initCosseratPlugin.cpp`. 
You could also add the following code in `config.h.in`:
```
namespace cosserat
{
	constexpr const char* MODULE_NAME = "@PROJECT_NAME@";
	constexpr const char* MODULE_VERSION = "@PROJECT_VERSION@";
}
```
to use the variables in `initCosseratPlugin.cpp`.

And I don't know what to do with `PythonEnvironment::addPythonModulePathsForPluginsByName("CosseratPlugin");`

**Comment by damienmarchal** [comment]
> you should also have a look in `initCosseratPlugin.cpp`. You could also add the following code in `config.h.in`:
> 
> ```
> namespace cosserat
> {
> 	constexpr const char* MODULE_NAME = "@PROJECT_NAME@";
> 	constexpr const char* MODULE_VERSION = "@PROJECT_VERSION@";
> }
> ```
> 
> to use the variables in `initCosseratPlugin.cpp`.
> 
> And I don't know what to do with `PythonEnvironment::addPythonModulePathsForPluginsByName("CosseratPlugin");`

I don't know either, it depend what is the status of the deployement done by Guillaume. That line was to add to the python interpreter the path for python module that is just "next" to the loaded plugin. 

The idea was to be able to ship a plugin with this kind of structure:
```console
CosseratPlugin/
     - libCosseratPlugin.so (or .dll)
     - python3/cosserat
```
So that ones can do "import cosserat" easily.

**Comment by alxbilger** [comment]
The only references to the old name `CosseratPlugin` are:

- `import Sofa.CosseratPlugin` in some scenes
- `PythonEnvironment::addPythonModulePathsForPluginsByName("CosseratPlugin");`
- `PYBIND11_MODULE(CosseratPlugin, m)`

and I don't know what to do with them. @damienmarchal ?

After that, the PR will be ready

**Pull request closed without merging**